tests/release/e2e-test-default-with-deployment.go

```diff
@@ -167,11 +167,12 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1199]test-release-e2e-with-deploy
 		if err != nil {
 			Expect(err.Error()).To(ContainSubstring("Reference does not exist"))
 		}
-		// TODO: Uncomment this lines once: https://issues.redhat.com/browse/RHTAPBUGS-409 is solved
-		/*if !CurrentSpecReport().Failed() {
-			Expect(fw.AsKubeAdmin.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+		if !CurrentSpecReport().Failed() {
+			// Expect(fw.AsKubeAdmin.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+			// Expect(fw.AsKubeAdmin.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
```

Comment:
What are these commented-out lines for? Don't we want to delete the devNamespace as well?

Reply:
yes, 
I had to remove those commented lines.
removed

Reply:
devNamespace will be deleted in 

`Expect(fw.SandboxController.DeleteUserSignup(fw.UserName)).NotTo(BeFalse())`

Reply:
ok, looks good, thanks.